frame/bridge/ethereum/relay/src/lib.rs

```diff
@@ -693,6 +662,22 @@ impl HeaderThing for EthereumHeaderThing {
 	}
 }
 
+#[derive(Clone, Default, PartialEq, Encode, Decode, RuntimeDebug)]
+pub struct ConfirmedEthereumHeaderInfo {
+	/// Ethereum Block Number
+	pub header: EthereumHeader,
+	/// MMR root of all previous headers.
+	pub mmr_root: H256,
+}
+
+#[cfg_attr(any(feature = "deserialize", test), derive(serde::Deserialize))]
+#[derive(Clone, Default, PartialEq, Encode, Decode, RuntimeDebug)]
+pub struct MMRProof {
+	pub member_leaf_index: u64,
+	pub last_leaf_index: u64,
+	pub proof: Vec<H256>
+}
```

Comment:
Should update this new struct in `types.json` together

Reply:
Updated